server/src/main/java/org/opensearch/common/BytesRefUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common;
+
+import org.apache.lucene.util.BytesRef;
+
+/**
+ * A set of utilities for {@link BytesRef}
+ *
+ * @opensearch.internal
+ */
+public final class BytesRefUtils {
```

Comment:
Looks like a good candidate for a unit test (sorry if I missed it...a lot of file here :) )

Reply:
+1

Reply:
Already there: BytesRefUtilsTests.java (opensearch-core)

Reply:
Ok, now I'm more confused. The class `org.opensearch.core.util.BytesRefUtils` already exists, and this PR adds a new `org.opensearch.common.BytesRefUtils` class? And there is no unit test for the newly added `org.opensearch.common.BytesRefUtils.bytesToLong` method?

Reply:
Oh sorry, for confusion, my bad: `BytesRefUtils` was extracted from `Numbers` so the later could be moved to common, I genuinely believed that `BytesRefUtilsTest` tests that. `BytesRefUtils` is not used in this change at all, it was extracted in order to not bring Lucene dependencies.

Reply:
Any concerns with moving this method to the existing `org.opensearch.core.util.BytesRefUtils` class to avoid a duplicate class name?

Reply:
> Any concerns with moving this method to the existing `org.opensearch.core.util.BytesRefUtils` class to avoid a duplicate class name?

No concerns, I just missed the fact we already had one, thanks @andrross! 
Updated and test added